packages/react-components/src/organisms/GroupTools.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import css from '@emotion/css';
+import { GroupResponse } from '@asap-hub/model';
+
+import { Card, Headline3, Paragraph, Link } from '../atoms';
+import { getIconFromUrl } from '../utils';
+import { perRem } from '../pixels';
+
+const BUTTON_SPACING = 28 / perRem;
+const buttons = css({
+  display: 'flex',
+  flexFlow: 'wrap',
+  marginTop: `${18 / perRem}em`,
+  width: `calc(100% + ${BUTTON_SPACING}em)`,
+  marginRight: `-${BUTTON_SPACING}em`,
+  '> a': {
```

Comment:
Can these children be `li`s with 'sandboxed' styles instead of reaching into `Link`? (and the `div` of course `ul`)

Reply:
(I realize we can't reset `marginTop` then, but seems we can work with `marginTop` and reduce our own)